github/github-release/github-release.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: github-release
```

Comment:
suggest create-github-release as a name here.  It might be that at some point we also want a remove-github-release or draft-github-release.

Reply:
Okay. will change this.